WordPress/Classes/ViewRelated/Views/BottomSheet/Demo/BottomSheetDemoViewController.swift

```diff
@@ -0,0 +1,45 @@
+
+import UIKit
+
+// MARK: - BottomSheetDemoViewController
+
+class BottomSheetDemoViewController: UINavigationController {
+
+    let resultsViewController = LocationResultsTableViewController()
```

Comment:
If I understand correctly it won't make any actual difference because it is used only within the context of this type, but would it makes sense to declare it as private? That way the declaration signals what to expect in terms of access clearly and without ambiguity